ckanext/dcat/profiles.py

```diff
@@ -740,6 +793,12 @@ def parse_dataset(self, dataset_dict, dataset_ref):
         # License
         if 'license_id' not in dataset_dict:
             dataset_dict['license_id'] = self._license(dataset_ref)
+        
+        # Source Catalog
+        if catalog_src is not None:
```

Comment:
if catalog_src is not None and asbool(config.get(DCAT_EXPOSE_SUBCATALOGS):

Reply:
We are simply storing the source catalog info here. 
We may want to have them stored anyway, even if those info are not exposed during serialization.

Reply:
I really want to make optional, defaulting to False, the storing of these fields. If you want you can add a second config option (`ckanext.dcat.store_subcatalogs` or something) that only triggers the storing. So the fields we'll be stored if `ckanext.dcat.expose_subcatalogs` or `ckanext.dcat.store_subcatalogs` are True